content/apps/seismic-risk/index.en.md

```diff
@@ -31,6 +31,9 @@ team:
 partners:
   - mkbt
 
+aliases:  
+  - /content/apps/seismic-alert/
```

Comment:
Aliases [need to be prefixed](https://gohugo.io/content-management/urls/#example-aliases-in-multilingual) with the language code.

```suggestion
  - /en/content/apps/seismic-alert/
```